src/relentless/simulate/lammps.py

```diff
@@ -1641,3 +1643,10 @@ def _initialize_engine(self, sim):
     _EnsembleAverage = EnsembleAverage
     _Record = Record
     _WriteTrajectory = WriteTrajectory
+
+
+def to_int(a):
```

Comment:
Let's make this explicitly private. These modules are really part of the public API, but I don't want to promise this function will always work. You should also add a doc string.
```suggestion
def _to_int(a):
```